FakeItEasy.sln

```diff
@@ -13,14 +13,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		build.cmd = build.cmd
 		build.ps1 = build.ps1
 		build.sh = build.sh
-		CONTRIBUTING.md = CONTRIBUTING.md
 		Directory.Build.props = Directory.Build.props
-		how_to_build.md = how_to_build.md
 		License.txt = License.txt
-		mkdocs.yml = mkdocs.yml
 		README.md = README.md
-		requirements.in = requirements.in
-		requirements.txt = requirements.txt

```

Comment:
I realize it's going to conflict with your PR. I'll fix it after yours is merged